server/src/lsp-server.spec.ts

```diff
@@ -50,10 +50,15 @@ describe('completion', () => {
             textDocument: doc,
             position: pos
         }) as TSCompletionItem[];
-        assert.isTrue(proposals.length > 800, String(proposals.length));
-        const item = proposals.filter(i => i.label === 'addEventListener')[0];
+        assert.isAtLeast(proposals.length, 800);
+        const item = proposals.find(i => i.label === 'addEventListener');
+        if (!item) {
+            assert.fail('missing addEventListener completion');
+            return;
+        }
```

Comment:
Or:
```suggestion
        assert.isDefined(item, 'missing addEventListener completion');
        if (!item) {
            return;
        }
```
(annoying that TS doesn't understand this automatically ;))